plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -280,8 +293,19 @@ def create_or_update_project(self, project_name, namespace, options):
             })
             if options['initialize_with_readme']:
                 project_options['initialize_with_readme'] = options['initialize_with_readme']
+
+            if options['default_branch']:
```

Comment:
Shouldn't this be indented to be inside the above `if`?

Reply:
I'm sorry i don't understand.  default_branch can't work if project already exist, same as initialize_with_readme.